okfn-brasil/serenata-de-amor
# Cleanup: remove unused ceap-datasets.html from Jarbas

**What is the purpose of this Pull Request?**

The `/static/ceap-dataset.html` file from Jarbas was not updated, and is not served anymore since we moved form S3 to Spaces. As this was a barely unnused page, I propose it's removal.

**What was done to achieve this purpose?**

* Removed it from Jarbas drawler menu
* Removed it's references in the i18n module
* Removed the command that downloaded it and converted from `.md` to `.html`
* Removed the dependencies from `requirements.txt`
* Removed the needed settings from `.env.sample` and from the docs
* Removed references to it in the dcos

**How to test if it really works?**

Build `django` and `elm` containers locally, access `/layers/` and check there's no _About the dataset_ link in the drawler menu (the one that is hidden in the left, accessible through the hamburger icon):

```diff
@@ -9,7 +9,9 @@ services:
       - ./rosie:/code
 
   django:
-    image: serenata/django
+    # image: serenata/django
+    build:
+      context: .
     depends_on:
       - elm
       - postgres
@@ -40,7 +42,10 @@ services:
 
   elm:
     command: ["npm", "run", "watch"]
-    image: serenata/elm
+    # image: serenata/elm
+    build:
+      dockerfile: Dockerfile-elm
+      context: .
     volumes:
       - ./jarbas:/code/jarbas
     hostname: elm
```

**Who can help reviewing it?**

@anaschwendler @jtemporal @irio @cabral
